code/obj/medicaldiagnosis.dm

```diff
@@ -0,0 +1,68 @@
+/obj/item/medical/medicaldiagnosis
+	name = "UM EXCUSE ME WHY ARE YOU SEEING THIS PLEASE MAKE AN ISSUE AAAAA"
+	desc = "*scream"
+	icon_state = "stethoscope" // todo
+	inhand_image_icon = "stethoscope" // nuke this awful sprite later
+	icon = 'icons/obj/medicaldiagnosis.dmi' // NOO YOU CAN'T JUST FILL YOUR CODE WITH TODO MARKERS YOU NEED TO ACTUALLY DO SPRITING AND NOT JUST MAKE CODERSPRITES NOO
+// haha procrastination machine go brrr
+/obj/item/medical/medicaldiagnosis/stethoscope
+	name = "stethoscope"
+	desc = "a disc-shaped resonator attached to two earpieces for figuring out if someone has consumption or is simply suffering from the vapors."
+	attack(mob/living/carbon/human/M as mob, mob/user as mob)
+		if(user.a_intent == "help")
+			if ((user.bioHolder.HasEffect("clumsy") || user.get_brain_damage() >= 60) && prob(50) || (!user.traitHolder.hasTrait("training_medical") && prob(10)) || prob(2))
+				user.visible_message("<span style=\"color:red\"><b>[user]</b> stabs themself in the ears with [src]!</span>", "<span style=\"color:red\">You stab yourself in the ears with [src]!</span>")
+				user.apply_sonic_stun(0, 0, 0, 0, 0, 12, 6)
+				take_bleeding_damage(user, user, 15)
+			else
+				/* Order for this stuff:
+				Lung failure
+				Sarin/Strychnine
+				Severe Oxygen Loss
+				Lung damage
+				Other Poisons
+				Drugs
+				Light oxygen loss
+				Anything else
+				*/
+				user.visible_message("<span style=\"color:blue\"><b>[user]</b> puts [src] to [(user != M) ? "[M]'s" : "their"] chest.</span>", "<span style=\"color:blue\">You put [src] to [(user != M) ? "[M]'s" : "your"] chest and begin listening.</span>")
+				if(do_after(user, (user.traitHolder.hasTrait("training_medical") ? 2 SECONDS : 4 SECONDS)) && !(user.getStatusDuration("stunned") || user.getStatusDuration("weakened") || user.getStatusDuration("paralysis") > 0 || !isalive(user) || user.restrained()))
+					if(isalive(M) == false)
```

Comment:
Should probably have a case for the patient having no lungs at all

Reply:
resolved in next commit